spring-cloud-skipper-shell-commands/src/main/java/org/springframework/cloud/skipper/shell/support/SkipperJLineShellComponent.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.skipper.shell.support;
+
+import org.springframework.shell.core.CommandResult;
+import org.springframework.shell.core.JLineShellComponent;
+import org.springframework.util.Assert;
+
+/**
+ * @author Mark Pollack
+ */
+// TODO enabling this gives errors on startup, investigate.
```

Comment:
Yea, that is what I thought.  One ends up in a bad state if any exception is thrown at startup inside the `target` command which is what I realize was going on now.  There is something brittle here... command like 'help' didn't even register in this case for some reason (I guess due to initialization ordering....)